### PR TITLE
Feature/color and global styles

### PR DESCRIPTION
**Pull request by 0xRowdy**
## GitHub Issue

[GitHub issue.](https://github.com/HausDAO/daohaus-monorepo/issues/143)

## Changes

Re-organize the styles and theme. Create a colors file, and create aliases for colors that map to utility to hold colors based on Radix Colors.

## Packages Added

NA

## Checks

Before making your PR, please check the following:

- [x] Critical lint errors are resolved
- [x] App runs locally
- [x] App builds locally (run the build command for *any impacted package* and check for any errors before the PR)

### CODE REVIEW

File: libs/ui/src/components/atoms/Switch/Switch.tsx [hunk 1]
```diff
@@ -73,7 +73,7 @@ export const Switch = React.forwardRef(
           onCheckedChange={onCheckedChange}
           disabled={disabled}
         >
-          <SwitchSlider className={classes} ref={ref} />
+          <SwitchSlider ref={ref} />
         </SwitchBase>
         <LabelContainer>
           <Label id={id} className={classes}>
```
Comment (0xRowdy) [inline libs/ui/src/components/atoms/Switch/Switch.tsx:79]: We should make a ticket to refactor Switch to use Label Component. 
I don't want to keep bloating this ticket.

File: libs/ui/src/theme/theme.ts [hunk 2]
```diff
@@ -1,167 +1,226 @@
...
   PrimaryLightButton,
   SecondaryDarkButton,
   SecondaryLightButton,
   TertiaryDarkButton,
   TertiaryLightButton,
-} from './component/button';
+} from './atoms/button';
+import { FieldTheme, FieldThemeDark } from './atoms/field';
 
 export const defaultDarkTheme: Theme = {
```
Comment (0xRowdy) [inline libs/ui/src/theme/theme.ts:24]: I would be interested in the idea of creating an atomic design folder structure for the theme?
I'll put togeather a hack md file to cover the pros & cons.

File: libs/ui/src/components/atoms/Switch/Switch.tsx (hunk 1 above)
Comment (jordanlesich) [reply libs/ui/src/components/atoms/Switch/Switch.tsx:79]: Agreed.

File: libs/ui/src/theme/theme.ts (hunk 2 above)
Comment (jordanlesich) [reply libs/ui/src/theme/theme.ts:24]: I like the rigour, but we're in ultra-budget mode now. Probably best to leave it for now.

File: libs/ui/src/components/atoms/Button/Button.styles.ts [hunk 3]
```diff
@@ -6,142 +6,118 @@ export const ButtonBase = styled.button`
   background-color: ${({ theme }) => theme.button.primary.bg};
   border: 0.1rem solid
     ${({ theme }: { theme: Theme }) => theme.button.primary.border};
+  border-radius: 0.4rem;
```
Comment (jordanlesich) [inline libs/ui/src/components/atoms/Button/Button.styles.ts:9]: Don't we have a const for this?

File: libs/ui/src/components/atoms/Button/Button.styles.ts [hunk 4]
```diff
@@ -152,27 +128,33 @@ export const ButtonBase = styled.button`
...
       height: 1.5rem;
       margin-right: 0.8rem;
+      width: 1.5rem;
     }
   }
+
   svg.icon-left {
     margin-right: 1rem;
   }
+
```
Comment (jordanlesich) [inline libs/ui/src/components/atoms/Button/Button.styles.ts:157]: This component looks much cleaner

**Review by jordanlesich — APPROVED** [review]
This is looking awesome. Super thorough. There's lots of work here. Thanks man!

**Comment by jordanlesich** [comment]
I changed the tooltip color as I noticed that it was secondary in the designs. I also added a generic secondary to the main theme. I went through everything in Storybook. Looking good.

**Pull request merged**